docker/dev/config_docker.json

```diff
@@ -0,0 +1,90 @@
+{
+    "LogSettings": {
+        "ConsoleEnable": true,
+        "ConsoleLevel": "INFO",
+        "FileEnable": true,
+        "FileLevel": "INFO",
+        "FileFormat": "",
+        "FileLocation": ""
+    },
+    "ServiceSettings": {
+        "SiteName": "Mattermost",
+        "Mode" : "dev",
+		"AllowTesting" : true,
+        "UseSSL": false,
+        "Port": "80",
+        "Version": "developer",
+        "Shards": {
+        },
+        "InviteSalt": "gxHVDcKUyP2y1eiyW8S8na1UYQAfq6J6",
+        "PublicLinkSalt": "TO3pTyXIZzwHiwyZgGql7lM7DG3zeId4",
+        "ResetSalt": "IPxFzSfnDFsNsRafZxz8NaYqFKhf9y2t",
+        "AnalyticsUrl": "",
+        "UseLocalStorage": true,
+        "StorageDirectory": "/mattermost/data/"
```

Comment:
FYI I think this file is missing some config changes, but since it's dev we can fix afterwards.